docs/src/osdk/reference/manifest.md

```diff
@@ -32,58 +32,103 @@ Below, you will find a comprehensive version of
 the available configuration options in the manifest.
 
 ```toml
-kcmd_args = ["init=/bin/busybox", "path=/usr/local/bin"] # <1>
-init_args = ["sh", "-l"] # <2>
-initramfs="./build/initramfs.cpio.gz" # <3>
-
-[boot]
-loader = "grub" # <4>
-protocol = "multiboot2" # <5>
-grub-mkrescue = "/usr/bin/grub-mkrescue" # <6>
-ovmf = "/usr/bin/ovmf" # <7>
-
-[qemu]
-path = "/usr/bin/qemu-system-x86_64" # <8>
-machine = "q35" # <9>
-args = [ # <10>
-    "-enable-kvm",
-    "-m 2G", 
-    "-device virtio-keyboard-pci,disable-legacy=on,disable-modern=off"
-] 
+[project] # <1>
+type = "kernel"
+
+[run] # <2>
+kcmd_args = ["init=/bin/busybox", "path=/usr/local/bin"] # <3>
+init_args = ["sh", "-l"] # <4>
+initramfs="./build/initramfs.cpio.gz" # <5>
```

Comment:
Grouping these three lines seems to be a good idea. A candidate name is `kernel_init`, which means the information provided for kernel initialization.

Reply:
A confusing name to me, as `init` already represents the init process. Doing `kernel_init.initproc` really lengthens the field name.